playbooks/common/openshift-cluster/config.yml

```diff
@@ -8,10 +8,10 @@
   tags:
   - always
   gather_facts: no
-  tasks:
-  - include_role:
-      name: openshift_excluder
-      tasks_from: disable
+  roles:
+  - role: openshift_excluder
+    r_openshift_excluder_action: disable
+    r_openshift_excluder_service_type: "{{ openshift.common.service_type }}"
 
```

Comment:
I feel like every time we've reached for include_role we've ended up creating problems for ourselves like those @ingvagabund mentioned and @mtnbikenc has previously hi-lighted. I like calling the role with a control variable as proposed here.